src/python/ensembl/ncbi_taxonomy/models.py

```diff
@@ -0,0 +1,69 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    Index,
+    join,
+    Table,
+)
+from sqlalchemy.dialects.mysql import (
+    INTEGER,
+    TINYINT,
+    VARCHAR,
+    CHAR,
+)
+from sqlalchemy.orm import (
+    relationship,
+    column_property,
+)
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+metadata = Base.metadata
```

Comment:
Is `metadata` needed here?

Reply:
Oops, not anymore, I had another piece of code I removed and forgot this bit :+1: